packages/web/src/common/store/cache/collections/createPlaylistSaga.ts

```diff
@@ -58,6 +58,9 @@ function* createPlaylistWorker(
     | typeof cacheCollectionsActions.createPlaylist
   >
 ) {
+  // Return early if this is an album
+  if (action.isAlbum) return
```

Comment:
If we get here, is it an error? Maybe worth reporting?

Reply:
no it's not actually, the "createPlaylistSaga" and "createAlbumSaga" both run when CREATE action is made, but only one of them run based on this criteria